app/components/viral/dropdown_component.rb

```diff
@@ -20,6 +20,7 @@ def initialize(label: nil, icon: nil, caret: false, trigger: TRIGGER_DEFAULT,
       @trigger = TRIGGER_MAPPINGS[trigger]
 
       @system_arguments = system_arguments
+      @system_arguments[:id] ||= "dd-#{SecureRandom.hex(4)}"
```

Comment:
Could this possibly lead to an issue of duplicate ids?

Reply:
I increased it to 10 so I think that there should any duplicates. Thanks

Reply:
I increased it to 10 so I think that there should any duplicates. Thanks